src/components/interfaces/MOBILE_API.xml

```diff
@@ -5907,7 +5907,7 @@
 
   <function name="OnWayPointChange" functionID="OnWayPointChangeID" messagetype="notification">
     <description>Notification which provides the entire LocationDetails when there is a change to any waypoints or destination.</description>
-    <param name="wayPoints" type="LocationDetails" mandatory="true" array="true" minsize="1" maxsize="10">
+    <param name="wayPoints" type="LocationDetails" mandatory="true" array="true" minsize="0" maxsize="10">

```

Comment:
@Itileda what about HMI_API? Probably you should change `minsize` there accordingly

Reply:
@AKalinich-Luxoft , Thanks! Done for both.